MoreLinq.Test/MoreLinq.Test.csproj

```diff
@@ -62,6 +62,7 @@
       <Compile Include="..\MoreLinq\Debug.cs" Link="Debug.cs" />
       <Compile Include="..\MoreLinq\Disposable.cs" Link="Disposable.cs" />
       <Compile Include="..\MoreLinq\Reactive\Subject.cs" Link="Subject.cs" />
+      <Compile Include="Extensions.cs" />
```

Comment:
Why have `EnableDefaultCompileItems` turned off and then manually insert items? If there are items to exclude from compilation, better to do `<Compile Remove="xxx.cs"/>`. (Probably a separate PR to fix, but comment here to put on your todo list.)